### PR TITLE
[#140641719] Styleguide: Tabs

### PR DESCRIPTION
**Pull request by saurabhsharma722**
@zamakkat RFR
http://luxola-staging-charon.herokuapp.com/style-guide

### CODE REVIEW

File: app/assets/stylesheets/sephora_style_guide/components/_tabs.scss [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    line-height: 30px;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $black;
+    }
+  }
+}
+
+.tab-content {
```
Comment (zamakkat) [inline app/assets/stylesheets/sephora_style_guide/components/_tabs.scss]: Shouldn't this be applied only to `.tab-contant-default`? Cause it affects the other tabs too and I am not sure that's what you want

File: app/views/sephora_style_guide/home/_tabs.slim [hunk 2]
```diff
@@ -0,0 +1,79 @@
+article#tabs
+  h2.h2 Tabs
+
+  section
+    h3.h3 Default
+    .tabs
```
Comment (zamakkat) [inline app/views/sephora_style_guide/home/_tabs.slim]: Any reason to use the container?

File: app/assets/stylesheets/sephora_style_guide/components/_tabs.scss (hunk 1 above)
Comment (saurabhsharma722) [inline app/assets/stylesheets/sephora_style_guide/components/_tabs.scss]: these style related to content of tab which seems common is both kind of tabs, that why I prefer it to add it here.

File: app/views/sephora_style_guide/home/_tabs.slim (hunk 2 above)
Comment (saurabhsharma722) [inline app/views/sephora_style_guide/home/_tabs.slim]: @zamakkat yep, when window size is reduce and tabs should turn to scrollable tabs but also we don't want to show scrollbar show I used `container` to fix such kind of requirements .

Comment (zamakkat) [inline app/views/sephora_style_guide/home/_tabs.slim]: oh OK then. Thanks for the explanation 👍

File: app/assets/stylesheets/sephora_style_guide/components/_tabs.scss (hunk 1 above)
Comment (zamakkat) [inline app/assets/stylesheets/sephora_style_guide/components/_tabs.scss]: As mentioned in chat, maybe move the styles to be under `.tab-content-default` so we can de-couple functionality from styles. Also consider merging some of the rules if appropriate.

File: app/views/sephora_style_guide/home/_tabs.slim [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      ul.nav.nav-tabs.nav-tabs-primary role='tablist'
+        li.nav-item
+          a.nav-link.active data-toggle='tab' href='#ship-to-address' role='tab' Ship to address
+        li.nav-item
+          a.nav-link data-toggle='tab' href='#self-pick-up-at-station' role='tab' Self pick up at station $x
+        li.nav-item
+          a.nav-link data-toggle='tab' href='#self-pick-up-at-bumbox' role='tab' Self pick up at bumbox $x
+
+    .tab-content.nav-tabs-primary
```
Comment (zamakkat) [inline app/views/sephora_style_guide/home/_tabs.slim]: Do you need the `nav-tabs-primary` here?

**Comment by saurabhsharma722** [comment]
**MERGED**

**Pull request closed without merging**